RELEASE-NOTES.txt

```diff
@@ -25,6 +25,8 @@ Bug
 Improvement
 
     [OAK-9261] - Upgrade Apache Solr to 8.6.3 and remove Embedded Solr Server
+    [OAK-9176] - Fix for sweep missing _bc (branch commit) entries in some cases - plus sweep2 introduced to clean up repository accordingly.
+                 PS: use -Doak.documentMK.disableSweep2=true to disable sweep2 in case it should be skipped explicitly
```

Comment:
This is usually done by the person who creates the release based on information in JIRA.
I think it would be better if additional information is added further up in the release notes.